recipes-multimedia/bento4/bento4_git.bb

```diff
@@ -0,0 +1,36 @@
+SUMMARY = "Bento4 is a C++ class library and tools designed to read and write ISO-MP4 files."
+HOMEPAGE = "https://github.com/axiomatic-systems/Bento4" 
+
+LICENSE = "GPL-2.0-only"
```

Comment:
> Each version is given a distinguishing version number.  If the Program
specifies a version number of this License which applies to it and "any
later version", you have the option of following the terms and conditions
either of that version or of any later version published by the Free
Software Foundation.  If the Program does not specify a version number of
this License, you may choose any version ever published by the Free Software
Foundation.

Dual license, but I think we can stay with GPL-2.0-or-later

Reply:
https://github.com/axiomatic-systems/Bento4/blob/master/Documents/LICENSE.txt

looked like GPL-2.0-only to me

Reply:
> or of any later version published by the Free Software Foundation
?

Reply:
> . If the Program specifies a version number of this License which applies to it and "any
> later version"

But that doesn't seem to be the case. I cant find a word about "any later version" in the given license https://github.com/axiomatic-systems/Bento4/blob/3dc8d58b0863b7058fc48a0d6583ffe2ae282ece/Documents/LICENSE.txt#L19

Reply:
Oh ok, now I get it. Sorry for trouble ;)

I need to fix some of my recipes since I was mistaken that section "any later version" is enough to make it GPL-2.0-or-later. Well happens.

p.s. GPL-2.0-or-later as template do not have or later in header, this is some how confusing.